internal/cli/root/atlas/builder.go

```diff
@@ -88,10 +88,6 @@ func Builder(profile *string) *cobra.Command {
 			"toc": "true",
 		},
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			if config.Service() == "" {
```

Comment:
can you explain a bit more about this removal?

Reply:
When running `atlas config ls` without any configuration, the command was listing the default profile with `service = cloud`. it took me a while to understand why and the reason was this piece of code that was copied/pasted from the atlas command.

Reply:
[nit/idea] Regardless of the decision, should we output to stdout that we're assuming cloud if the value was not set? Would improve the UX and be make the flow more predictable.

The tradeoff is being more verbose/maybe users don't care.

Reply:
this was kind of requested, we didn't use to assume and force people to do `MONGODB_ATLAS_SERVICE=cloud` and people hated it and asked if they were using an atlas command why they need to specify the service since  is a given by the name of the command 🤷

Reply:
> I expect that to be successful without having to explicitly set the service, and the reason behind setting the service here

@gssbzn why? given that the cli can be used with both atlas and atlas for gov, this expectation doesn’t seem correct

> [nit/idea] Regardless of the decision, should we output to stdout that we're assuming cloud if the value was not set? Would improve the UX and be make the flow more predictable
yes, I agree that if we decide to automatically assign the service, we should print a warning message

Reply:
Sorry I haven’t seen your response in the thread.

Reply:
I have updated my changes. Now I am not setting the service only when running the config and completion commands. I am also printing a message when we set the service.


```
./bin/atlas project list
service not set. Atlas CLI used 'cloud'

Error: this action requires authentication

To log in using your Atlas username and password, run: atlas auth login
To set credentials using API keys, run: atlas config init
```

```
./bin/mongocli atlas cluster ls
service not set. MongoCLI used 'cloud'

Error: this action requires authentication

To log in using your Atlas username and password, run: mongocli auth login
To set credentials using API keys, run: mongocli config
```

Reply:
@andreaangiolillo this request predates atlas gov, also we are promising people a smooth transition from mongocli to atlas cli, not fulfilling this expectation is a breaking change.

Message should be review by product, `cloud` is a confusing terms from the time when originally mogndob cli would also include things like realm and not atlas only so people may not understand what we mean by `cloud` here

Also notice this warning message should go to stderr otherwise it's a breaking change and you'll break any tool parsing the output